src/jacktrip_globals.cpp

```diff
@@ -146,7 +146,6 @@ void setRealtimeProcessPriority()
 #ifdef __UBUNTU__
     priority = 95;  // anything higher is silently ignored by Ubuntu 18.04
 #endif
-    priority = 3;
```

Comment:
We should investigate this.

Reply:
For every Linux priority was set to 3. I don't think that's what you expect if you use this flag. So I just set it to 50 now. It's below jack and the audio threads. So that shouldn't interfere.

Reply:
I agree that the code was not clear at all (it was a mix of layers from different times). But setting the priority to 3 is the expected behavior. This priority should be lower than the priority of jack callbacks. Ideally I think it should be calculated directly from the jack callback priority which can be obtaind with jack API.

One reason why all this priority handling code is quite messy is that there were discussions and different attempts to improve jacktrip performance but this work hasn't been finished. There are still open questions and probably some disagreement.

Reply:
I pushed a commit that sets the udp rt priority 10 lower than the jack audio clients' priority. It's only tested with Linux.

Reply:
Well, on my Debian at least, jack sets the callback RT priority to 5 by default. That will result in a negative value for the priority although the valid range is 1..99 (if I remember correctly). That's where 3 came from btw. I think `jack_callback_priority - 1` should be fine. We don't whant to block jack callbacks and even one step lower is ok. Even more accurate way would be to check `sched_get_priority_min()` and use it as a low limit.

Reply:
Interesting. I was already surprised that the default for the jack server is 20 on Fedora.

Reply:
So it's -1 now.

Reply:
Is this okay or should I add a lower boundary too?

Reply:
If the question is about that UBUNTU line, please take it out. I put that there in 2018 when we still used a scary maximum.

Reply:
Yes, I removed that and modified the code so it sets the udprt priority 1 lower than the audio threads. But maybe we also need to set a lower boundary for the case that the audio thread priority was set to 0… A possible but rare case.
I think I will do it, because safety first.

Reply:
Done